int64 id
int64 v1
int64 v2
float32 length
geometry_msgs/Vector3 direction
bool is_boundary
bool is_walkable

#direction is defined from v1 to v2
